internal/backend/db/dbgorm/orgs.go

```diff
@@ -141,35 +175,50 @@ func (gdb *gormdb) RemoveOrgMember(ctx context.Context, oid sdktypes.OrgID, uid
 	)
 }
 
-func (gdb *gormdb) IsOrgMember(ctx context.Context, oid sdktypes.OrgID, uid sdktypes.UserID) (bool, error) {
+func (gdb *gormdb) GetOrgMemberStatus(ctx context.Context, oid sdktypes.OrgID, uid sdktypes.UserID) (sdktypes.OrgMemberStatus, error) {
 	if oid == authusers.DefaultOrg.ID() {
-		return uid == authusers.DefaultUser.ID(), nil
+		if uid == authusers.DefaultUser.ID() {
+			return sdktypes.OrgMemberStatusActive, nil
+		}
+
+		return sdktypes.OrgMemberStatusUnspecified, nil
```

Comment:
So the default org technically doesn't have members?

Reply:
i will turn the defaults into ordinary records in a future pr.